pyuvdata/uvdata/ms.py

```diff
@@ -399,17 +401,22 @@ def read_ms(
         # for measurement sets made with COTTER, this keyword is ITRF
         # instead of the epoch
         if epoch_string == "ITRF":
+            self.phase_center_frame = "icrs"
+            self.phase_center_epoch = 2000.0
```

Comment:
I can certainly change this to FK5 -- I did spend a little time looking at the code for cotter last night (when I was trying to chase down the small residual UVW errors), and it _seems_ like it things are in FK5 @ J2000, at least with the usage of [PAL](https://github.com/Starlink/pal) that I see. I'll go ahead and change it.

FWIW -- ITRF _is_ a supported frame in CASA, albeit one that I think is rarely (never?) used. Nevertheless, if one were to have coordinates in the ITRF frame, this fix will cause the reader to misinterpret the data (in particular, the UVWs and the phasing center position). Might be worth putting in a warning just to avoid this (likely very uncommon) corner-case.

Reply:
Added in a16d2352e370920d0859ce6843a8563f26c02744.

Reply:
yeah, maybe a warning is in order. We could also look at the history table to see if we could tell that it was a Cotter MS. We do that for uvfits because of a weirdness in Cotter uvfits files.